docs/general.md

```diff
@@ -0,0 +1,24 @@
+# About
```

Comment:
general comment: can you take a look at the consistency of SFT vs sft (capitalization) same goes for NFT vs nft